### PR TITLE
Roophee is on the board

### PR DESCRIPTION
**Pull request by Roophee**
Roophee join to the mock-repo. Write some words about myself. Add image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1700,4 +1700,12 @@ I'm very exciting to join the Kottans community :)
 
 Regards, [Dima](https://github.com/itro-dm)
 
+---
+# Hi, brothers in arms!
+
+I`mg glad to join this community and stsrt this trip!!!
```
Comment (miroslav-kolomiets) [inline README.md:1706]:
```suggestion
I'm glad to join this community and start this trip!!! 
```

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]

**Comment by babayK0** [comment]
@Roophee 
Resolve conflicts, please

File: README.md (hunk 1 above)
Comment (Roophee) [reply README.md]: @miroslav-kolomiets fixed typo, thanks

**Comment by Roophee** [comment]
@babayK0 done, thanks

**Review by babayK0 — APPROVED** [review]
@Roophee Nice catbird !
 :bird: + :cat: = :heart:

**Pull request merged**